ckanext/switzerland/dcat/profiles.py

```diff
@@ -489,7 +489,7 @@ def _add_distribution_to_graph(self, dataset_ref, resource_dict):
             (
                 distribution,
                 DCT.license,
-                Literal("NonCommercialAllowed-CommercialAllowed-ReferenceRequired"),
+                URIRef("http://dcat-ap.ch/vocabulary/licenses/terms_by"),
```

Comment:
This adds the `terms_by` license to all distributions, but after the migration the distributions should all have a correct license value and we should use that. I think we should add `("license", DCT.license, None, URIRef),` to the list `items` above, and delete the two statements that directly add the `DCT.rights` and `DCT.license` to the distribution graph.